pkg/common/util/hash_test.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GetSHA256Digest(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "file")
+	require.NoError(t, ioutil.WriteFile(path, []byte("some data"), 0600))
+	hash, err := GetSHA256Digest(path, -1)
+	require.NoError(t, err)
+	require.Equal(t, "1307990e6ba5ca145eb35e99182a9bec46531bc54ddf656a602c780fa0240dee", hash)
+}
+
+func Test_GetSHA256Digest_BelowLimit(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "file")
+	require.NoError(t, ioutil.WriteFile(path, []byte("some data"), 0600))
+	hash, err := GetSHA256Digest(path, 100)
+	require.NoError(t, err)
+	require.Equal(t, "1307990e6ba5ca145eb35e99182a9bec46531bc54ddf656a602c780fa0240dee", hash)
+}
+
+func Test_GetSHA256Digest_AboveLimit(t *testing.T) {
+	path := filepath.Join(t.TempDir(), "file")
+	require.NoError(t, ioutil.WriteFile(path, []byte("some data"), 0600))
+	hash, err := GetSHA256Digest(path, 5)
+	require.Error(t, err)
```

Comment:
instead of require.Error, can you use: `spiretest.RequireErrorContains()` so we are sure we hit the error we wanted?